examples/pruner/job.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: batch/v2alpha1
+kind: ScheduledJob
+metadata:
+  name: image-prune
+spec:
+  schedule: 0/15 * * * ?
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: prune
+            image: openshift/origin:latest
+            args: [ "admin", "prune", "images", "--confirm"]
+          restartPolicy: OnFailure
```

Comment:
Does it need to be restarted on failure? Running once in 15 minutes seem sufficient even if it occasionally fails for some reason.